dev-docs/dev-deployment.md

```diff
@@ -39,9 +39,56 @@ Set up an IP for your deployment:
    Note that certbot is installed with `brew install letsencrypt` on macOS.
 1. Upload the certificate: `gcloud compute ssl-certificates create <yourname>-ingress --certificate <path-to-fullchain.pem> --private-key <path-to-key>`. When the time comes to renew the certificate, simply increment the name (e.g., <yourname>-ingress-1). 
 
+### Minikube
+
+If you want to run taskcluster on a kubernetes cluster on a server or virtual machine, minikube can be used.
+Follow the regular installation for minikube, however:
+
+1. Make sure to deploy a kubernetes cluster with at least 2 CPUs and 4GB of memory.
+2. Enable the nginx ingress on the minikube cluster.
+
 #### Troubleshooting:
 * Certbot error `[Errno 13] Permission denied: '/var/log/letsencrypt' Either run as root, or set --config-dir, --work-dir, and --logs-dir to writeable paths.` - do not run as root, but set the directories instead.
 
+### SSL through a reverse proxy
```

Comment:
Please move this up above Troubleshooting as well (since it's linked to Minikube)

Reply:
Like it is right now?